src/js/utils/__tests__/NetworkValidatorUtil-test.js

```diff
@@ -38,5 +38,9 @@ describe("NetworkValidatorUtil", function() {
       expect(NetworkValidatorUtil.isValidPort(65535)).toBe(true);
       expect(NetworkValidatorUtil.isValidPort(65536)).toBe(false);
     });
+
+    it("verifies that port 0 is valid", function() {
```

Comment:
Probably don't need this test since we already have it https://github.com/dcos/dcos-ui/pull/2321/files#diff-d085bc6b0af09b2f1baa5824f77f9a01R22 and https://github.com/dcos/dcos-ui/pull/2321/files#diff-d085bc6b0af09b2f1baa5824f77f9a01R32 but I won't argue with good coverage ;)